src/main/docker/env.bash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+if [[ "${DOCKER_HOST}" ]]; then
+  export HOST=$(echo ${DOCKER_HOST} | sed -e 's|^tcp\://||;s|:.*$||');
```

Comment:
What does DOCKER_HOST varaible from? It looks like regex is extracting host name from a tcp address.
Anyone that knows what and when DOCKER_HOST is set and used.

Reply:
I just took the env.bash from EI. I can remove the if statement.

Reply:
I can't find that in EI env.bash file:
https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/docker/env.bash
If its not used or we don't know what its intended to be used for, then remove that line the if state and keep only code that is in else case.

Reply:
Done